locales/en-US/app.ftl

```diff
@@ -568,6 +626,66 @@ TrackContextMenu--hide-other-screenshots-tracks = Hide other Screenshots tracks
 TrackContextMenu--hide-track = Hide “{ $trackName }”
 TrackContextMenu--show-all-tracks = Show all tracks
 
+## TransformNavigator
+## Navigator for the applied transforms in the Call Tree, Flame Graph, and Stack
+## Chart components.
+## To learn more about them, visit:
+## https://profiler.firefox.com/docs/#/./guide-filtering-call-trees?id=transforms
+
+# Root item in the transform navigator.
+# See: https://profiler.firefox.com/docs/#/./guide-filtering-call-trees?id=collapse
+# Variables:
+#   $item (String) - Name of the current thread. E.g.: Web Content.
+TransformNavigator--complete = Complete “{ $item }”
+
+# "Collapse resource" transform.
```

Comment:
Given your explanation, can we change these comments a bit? 

It could be part of the section comment, e.g. 

> These messages are displayed above the table once the user selects to apply a specific transformation function to a node in the call tree. It's the name of the function, followed by the node's name.